### PR TITLE
Corrections to bring tutorials up-to-date

### PR DESCRIPTION
**Pull request by arporter**

### CODE REVIEW

**Comment by arporter** [comment]
Happily there were very few updates required and everything works as expected. I note that we could do with expanding the tutorials to cover more advanced topics like handling implicit loops and local work arrays. However, I'll create a new issue for that.
Ready for review from anyone!

File: tutorial/practicals/nemo/2_nemo_profiling/README.md [hunk 1]
```diff
@@ -81,20 +84,25 @@ you have time at the end of this session.)
...
    If you examine the Makefile, you will see that PSyclone has been run with
    the `--profile invokes` option and it is this that causes the subroutine
-   to be instrumented for profiling.
+   to be instrumented for profiling:
+
+       $(PSYCLONE) --profile invokes -api nemo \
+                     -opsy psy.f90 -l output tra_adv_mod.F90
 
 3. Edit the Makefile so that the line invoking PSyclone uses the
    `--profile kernels` option instead. Rebuild (you'll need to do a
```
Comment (sergisiso) [inline tutorial/practicals/nemo/2_nemo_profiling/README.md:93]: There is 2 "--profile invokes" in the Makefile and only one is relevant for this. Maybe this should be more specific about which Makefile rule to update?

Also the relevant rule has a commented out version without the flag, can this be deleted?

**Review by sergisiso — CHANGES_REQUESTED** [review]
Looks good @arporter . I just made 2 minor comments.

File: tutorial/practicals/nemo/1_nemo_psyir/README.md [hunk 2]
```diff
@@ -157,9 +157,10 @@ PSyIR of that invoke. This will be a lot of output, beginning with:
     tra_adv
```
[the remark this answers is not in the document]
Comment (arporter) [reply tutorial/practicals/nemo/1_nemo_psyir/README.md:157]: Good idea, done.

File: tutorial/practicals/nemo/2_nemo_profiling/README.md (hunk 1 above)
Comment (arporter) [reply tutorial/practicals/nemo/2_nemo_profiling/README.md:93]: Good point - I hadn't quite realised we had so many rules that ran PSyclone in that file. The commented out version is for part 2.1 of the tutorial.

**Review by sergisiso — APPROVED** [review]
@arporter All looks good now. PR ready to merge.

**Pull request merged**